nservicebus/messaging/databus/index.md

```diff
@@ -79,6 +93,19 @@ Automatically removing these attachments can cause problems in many situations.
 * If the data bus feature is used when publishing an event to multiple subscribers, neither the publisher nor any specific subscribing endpoint can determine when all subscribers have successfully processed the message allowing the file to be cleaned up.
 * If message processing fails, it will be handled by the [recoverability feature](/nservicebus/recoverability/). This message can then be retried some period after that failure. The data bus files need to exist for that message to be re-processed correctly.
 
+## Alternatives
+
+- Use a different transport or a different transport tier
+- Message Compression: Use message body compression which works well on text-based payloads like XML and Json or any payload (text or binary) that contains repetitive data
+  - [Message mutator example demonstrating message body compression](/samples/messagemutators/)
+- Stream-based properties: The [Handling large stream properties via pipeline](/samples/pipeline/stream-properties/) sample demonstrates a purely stream-based approach (rather than loading the full payload into memory) implemented by leveraging the NServiceBus pipeline.
+- Binary Serializer: A binary serializer is more efficient and most serializers can be added with a few lines of code
```

Comment:
> most serializers can be added with a few lines of code

@soujay what does this mean? That other serializers can be created, or that other serializers can be used, with a few lines of code?

Reply:
@soujay ping

Reply:
@adamralph 

Sorry, that I missed responding to the pings earlier. I couldn't find an email notification in my inbox.

>Binary Serializer: A binary serializer is more efficient and most serializers can be added with a few lines of code
 
 My understanding was that using a binary serializer is more efficient and most of the binary serializers can be easily implemented with a few lines of code.

Reply:
Thanks @soujay. I've made further edits to https://github.com/Particular/docs.particular.net/pull/6199